Integrations/integration-jira.yml

```diff
@@ -12,7 +13,10 @@ detaileddescription: |
   This will fetch for first time all tickets in your system (include past) that are in open status and in project soc, after first run, it will create incident only for new tickets,
   If you wish to start in first time from a specific position, use "Issue index to start fetching incidents from" param.
   
-  you can also fetch incidents by time (by issues created field) by turning on the  "Use created field to fetch incidents" field, instead of using id's.  
+  you can also fetch incidents by time (by issues created field) by turning on the  "Use created field to fetch incidents" field, instead of using id's.
+  Added new Authentication method : OAuth1.0.
+  To use OAuth1.0 you should folow this tutorial(https://developer.atlassian.com/cloud/jira/platform/jira-rest-api-oauth-authentication/) till you get Access Token.
```

Comment:
write this in markdown style
It will appear much better in the UI.
And the you can have links as well